src/main/java/org/apache/commons/text/similarity/IntersectionSimilarity.java

```diff
@@ -125,12 +125,7 @@ int uniqueElementSize() {
      * @return The intersection
      */
     private static <T> int getIntersection(final Set<T> setA, final Set<T> setB) {
-        int intersection = 0;
-        for (final T element : setA) {
-            if (setB.contains(element)) {
-                intersection++;
-            }
-        }
+        int intersection = (int) setA.stream().filter(setB::contains).count();
```

Comment:
Do we still need the local variable?

Reply:
Whichever way we go, we probably need to Javadoc the truncation (PR) or looping of ints (current master).

Reply:
inline.
meanwhile I have change your remarks.